tests/utils.py

```diff
@@ -175,6 +175,8 @@ async def new_consumer(c, group, fmt="avro", trail=""):
 
 
 def new_random_name(prefix: str) -> str:
+    # A hyphen is not a valid character for Avro identifiers. Use only the
+    # first 8 characters of the UUID.
     suffix = str(uuid.uuid4())[:8]
```

Comment:
Just a bit of documentation. It is odd at first glance that this is being done (should probably do something else, but I had enough changes on this PR)